src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -24,20 +21,18 @@
 public class ReportGenerator {
     private static final Logger logger = LogsManager.getLogger(ReportGenerator.class);
 
-    private static final DateFormat REPORT_NAME_FORMAT = new SimpleDateFormat("yyyy-MM-dd-HH-mm-ss");
-
     /**
      * Generates the authorship and commits JSON file for each repo in {@code configs} at {@code outputPath}, as
      * well as the summary JSON file of all the repos.
      */
     public static String generateReposReport(List<RepoConfiguration> configs, String outputPath) {
-        String reportName = generateReportName();
-        FileUtil.copyTemplate(outputPath, reportName);
-        Path templateLocation = Paths.get(outputPath, reportName,
+        String reportName = Paths.get(outputPath).getFileName().toString();
```

Comment:
The `reportName` is just the `outputPath` file name, there is no longer a need for this, and we can make `generateReposReport` void since the caller will already have the `reportName`.